CallAutomation_OutboundCalling/CallAutomation_OutboundCalling/Program.cs

```diff
@@ -22,20 +20,49 @@
 // Base url of the app
 var callbackUriHost = "<CALLBACK_URI_HOST_WITH_PROTOCOL>";
 
+// Your cognitive service endpoint
+var cognitiveServiceEndpoint = "<COGNITIVE_SERVICE_ENDPOINT>";
+
 // This will be set by fileStatus endpoints
 string recordingLocation = "";
 
 string recordingId = "";
 
+// text to play
+const string SpeechToTextVoice = "en-US-NancyNeural";
+const string MainMenu =
+    """ 
+    Hello this is Contoso Bank, we�re calling in regard to your appointment tomorrow 
+    at 9am to open a new account. Please confirm if this time is still suitable for you or if you would like to cancel. 
+    This call is recorded for quality purposes.
+    """;
+const string ConfirmedText = "Thank you for confirming your appointment tomorrow at 9am, we look forward to meeting with you.";
+const string CancelText = """
+Your appointment tomorrow at 9am has been cancelled. Please call the bank directly 
+if you would like to rebook for another date and time.
+""";
+const string CustomerQueryTimeout = "I�m sorry I didn�t receive a response, please try again.";
+const string NoResponse = "I didn't receive an input, we will go ahead and confirm your appointment. Goodbye";
+const string InvalidAudio = "I�m sorry, I didn�t understand your response, please try again.";
+const string ConfirmChoiceLabel = "Confirm";
+const string CancelChoiceLabel = "Cancel";
+const string RetryContext = "retry";
+
 CallAutomationClient callAutomationClient = new CallAutomationClient(acsConnectionString);
 var app = builder.Build();
 
 app.MapPost("/outboundCall", async (ILogger<Program> logger) =>
 {
     PhoneNumberIdentifier target = new PhoneNumberIdentifier(targetPhonenumber);
     PhoneNumberIdentifier caller = new PhoneNumberIdentifier(acsPhonenumber);
+    var callbackUri = new Uri(callbackUriHost + "/api/callbacks");
```

Comment:
the biggest customer painpoint here is sanity of callbackUriHost (lets say they add a '/' in the end, it will break the callback flow).. can we make it more robust to handle these? may be new Uri(callbackUriHost, "/api/callbacks") or Path.Combine?